metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/JoinBolt.java

```diff
@@ -141,11 +141,12 @@ public void execute(Tuple tuple) {
         collector.emit( "message"
                       , tuple
                       , new Values( key
-                                  , joinMessages(streamMessageMap)
+                                  , joinMessages(streamMessageMap, this.messageGetStrategy)
                                   )
                       );
         cache.invalidate(key);
-        collector.ack(tuple);
+        Tuple messageTuple = streamMessageMap.get("message:");
+        collector.ack(messageTuple);
```

Comment:
I don't believe so.  The tuple that comes straight from the SplitBolt (the "message" stream) is the only tuple that is anchored.  Tuples that go to EnrichmentBolts are not anchored.